modelservice/src/test/java/com/expedia/adaptivealerting/modelservice/test/ObjectMother.java

```diff
@@ -0,0 +1,64 @@
+package com.expedia.adaptivealerting.modelservice.test;
+
+import com.expedia.adaptivealerting.modelservice.entity.Metric;
+import com.expedia.adaptivealerting.modelservice.providers.graphite.GraphiteResult;
+import com.expedia.adaptivealerting.modelservice.providers.graphite.Tags;
+import com.expedia.adaptivealerting.modelservice.service.AnomalyRequest;
+import com.expedia.adaptivealerting.modelservice.spi.MetricSource;
+import com.expedia.adaptivealerting.modelservice.spi.MetricSourceResult;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class ObjectMother {
+    private static final ObjectMother MOM = new ObjectMother();
+
+    public static ObjectMother instance() {
+        return MOM;
+    }
+
+    private ObjectMother() {
+    }
+
+    public GraphiteResult[] getGraphiteData() {
+        GraphiteResult[] results = new GraphiteResult[1];
+        GraphiteResult result = new GraphiteResult();
+        String[][] datapoints = new String[2][2];
+        datapoints[0][0] = "78.0";
+        datapoints[0][1] = "1548829800";
+        datapoints[1][0] = "81.0";
+        datapoints[1][1] = "1548830400";
```

Comment:
Given that we defined our own `GraphiteResult` class here, I would not expect the data values and epoch timestamps to be strings. I would expect numerical values (say double/long). If Graphite expects string then map the numbers to strings at serialization time.